### PR TITLE
feat: added an oc cluster up version of script

### PR DESCRIPTION
**Pull request by austincunningham**
# What
Added a script for `oc cluster up`

# Why
not everybody uses minishift

# How
script is basically the same but except for how it obtains ip address , May need future changes to  improve persistence, and remove some of the minishift only commands.  

## Checklist:

- [ ] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member

### CODE REVIEW

File: setup-rhmap-oc-cluster-up.sh [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+# Progress : [####################--------------------] 50%
+printf "\rProgress : [${done// /#}${undone// /-}] ${precentage}%%"
+
+}
+
+
+# login to docker
+docker login
+
+# Create profile start minishift and find address
```
Comment (davidffrench) [inline setup-rhmap-oc-cluster-up.sh:33]: @austincunningham can just remove these lines entirely.

**Comment by davidffrench** [comment]
@austincunningham Is this a copy of the previous script or are there any additional changes?

**Comment by austincunningham** [comment]
@davidffrench its a copy but for how it obtains the ip address. Can remove the unwanted lines in further pr's

**Comment by davidffrench** [comment]
@austincunningham Why not use the same script with a conditional rather than having to maintain 2 scripts?

**Comment by austincunningham** [comment]
@davidffrench Good Idea, will close off in favor of one scipt

**Pull request closed without merging**